roles/tpa_single_node/meta/argument_specs.yml

```diff
@@ -0,0 +1,398 @@
+        type: "str"
+        version_added: "1.2.0"
+        default: "registry.redhat.io/rhtpa/rhtpa-guac-rhel9:f0688194637cc759052e02c350c38dbabc19484e"
+      tpa_single_node_base_hostname:
+        description: "The user name logging in to the registry to pull images."
+        type: "str"
+        version_added: "1.2.0"
+        default: "trustification"
+      tpa_single_node_rhel_host:
+        description: "Ip of the instance."
+        type: "str"
+        required: true
+        version_added: "1.2.0"
+      tpa_single_node_certificates_dir:
+        description: "Folder where to place the certificates to deploy on the instance."
+        type: "str"
+        version_added: "1.2.0"
+        default: "certs"
+      tpa_single_node_config_dir:
+        description: "Configuration directory on the instace."
+        type: "str"
+        version_added: "1.2.0"
+        default: "/etc/rhtpa"
+      tpa_single_node_kube_manifest_dir:
+        description: "Configuration directory on the instace containing the manifests."
+        type: "str"
+        version_added: "1.2.0"
+        default: "/etc/rhtpa/manifests"
+      tpa_single_node_namespace:
+        description: "Podman network namespace."
+        type: "str"
+        version_added: "1.2.0"
+        default: "trustification"
+      tpa_single_node_podman_network:
+        description: "Podman network name."
+        type: "str"
+        version_added: "1.2.0"
+        default: "tcnet"
+      tpa_single_node_systemd_directory:
+        description: "Folder where to store the systemd configurations files."
+        type: "str"
+        version_added: "1.2.0"
+        default: "/etc/systemd/system"
+      tpa_single_node_default_empty:
+        description: "Default empty value."
+        type: "str"
+        version_added: "1.2.0"
+        default: ""
+      tpa_single_node_pg_host:
+        description: "Host ip of the postgresql db instance. Readed from the TPA_PG_HOST env"
+        type: "str"
+        version_added: "1.2.0"
+        default: ""
+      tpa_single_node_pg_port:
+        description: "Port of the postgresql db instance."
+        type: "str"
+        version_added: "1.2.0"
+        default: "5432"
+      tpa_single_node_pg_db:
+        description: "DB name."
+        type: "str"
+        version_added: "1.2.0"
+        default: "guac"
+      tpa_single_node_pg_user:
+        description: "DB username."
+        type: "str"
+        version_added: "1.2.0"
+        default: "guac"
+      tpa_single_node_pg_user_passwd:
+        description: "DB password."
+        type: "str"
+        version_added: "1.2.0"
+        default: "guac1234"
+      tpa_single_node_pg_ssl_mode:
+        description: "DB SSL mode enabled/disabled."
+        type: "str"
+        version_added: "1.2.0"
+        default: "disable"
+      tpa_single_node_storage_type:
+        description: "Storage type s3/other."
+        type: "str"
+        version_added: "1.2.0"
+        default: "s3"
```

Comment:
Since event and oidc pointed to non-aws services, shall we default this for `minio`?